packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -835,6 +835,14 @@ export class DeltaManager
                 }
             }
 
+            if (to !== undefined && this.lastQueuedSequenceNumber >= to) {
+                // the client caught up while we were trying to fetch ops from storage
+                // bail out since we no longer need to request these ops
+                callback([]);
+                telemetryEvent.end({ deltasRetrievedTotal, requests });
```

Comment:
having some additional property here to differentiate this path vs. any other path might be useful later in debugging.